zproject/settings.py

```diff
@@ -190,6 +190,10 @@ def get_secret(key):
                     'POST_MIGRATION_CACHE_FLUSHING': False,
                     'ENABLE_FILE_LINKS': False,
                     'USE_WEBSOCKETS': True,
+                    'ENABLE_UGLIFY': True,
+                    'ENABLE_UGLIFY_COMPRESSION': True,
+                    'UGLIFY_ARGUMENTS': "--mangle --comments",
+                    'ENABLE_UGLIFY_SOURCEMAP': True
```

Comment:
I think we shouldn't need all these settings; `ENABLE_UGLIFY_SOURCEMAP` can probably be unconditionally true (is there a reason not to have it?); `ENABLE_UGLIFY` seems to be unused, etc.  I think we can probably get away with just a "UGLIFY_ARGUMENTS" option.